### DISCUSSION TITLE
launching issue

### DISCUSSION THREAD

**Discussion by onnelli** [VolcanicArts/VRCOSC#145] [Q/A]
hello, as the title suggests, im having an issue with vrcosc actually launching. ive ran a few fresh installs, cleared things with my antivirus, but ultimately im unable  to get vrcosc to properly launch. it doesn't appear in my task manager, and attempting to launch it multiple times doesnt help anything. any suggestions? thanks

**Comment by VolcanicArts** [comment]
Can you send your `runtime.log` file from `%appdata%/VRCOSC/logs` please?

**Reply by onnelli** [reply]
unfortunately, windows says such a directory does not exist

**Reply by VolcanicArts** [reply]
In that case VRCOSC hasn't installed correctly. Unless there's logs from the installer in `%localappdata%/SquirrelTemp` there's nothing I can do

**Reply by onnelli** [reply]
i appreciate what help you could offer. do you have any further suggestions, or will i just permanently get stuck with vrcosc not installing correctly?

**Reply by VolcanicArts** [reply]
You could try manually installing the .NET 6 runtime: https://dotnet.microsoft.com/en-us/download/dotnet/6.0
I think I remember that fixing this issue for a couple other people

**Reply by onnelli** [reply]
having installed the .NET 6 runtime, and reinstalling VRCOSC and restarting my pc, I checked the local folder of appdata instead of the roaming, and was able to return with this log from installation, and VRCOSC still not opening properly
[Squirrel.log](https://github.com/VolcanicArts/VRCOSC/files/12566572/Squirrel.log)

**Reply by VolcanicArts** [reply]
It looks like it's installing fine. If the installation is present in `%localappdata%/VRCOSC` but it's still not opening that could indicate that it's still missing dependencies that should be installed on your PC.
I think I also remember that installing Visual Studio (for whatever reason) fixes some dependency issues, so you can try that?
https://visualstudio.microsoft.com/downloads/

**Reply by onnelli** [reply]
Gave that a try, as well as a fresh PC restart and attempt again, still encountering the same issue of the application not opening

**Reply by VolcanicArts** [reply]
Then there's nothing else I can do. Squirrel isn't reporting anything wrong, and the app isn't opening enough to start reporting logs, so I'm almost certain it's a dependency issue, but either of those methods should've solved it if you were missing something critical

**Reply by onnelli** [reply]
Alrighty, if that's the case, then I appreciate you having done all you could!
